src/pro_create.cpp

```diff
@@ -599,7 +599,37 @@ void update_buildrepair(void)
         STOP_WORKING(CH);
       } else if (AFF_FLAGGED(CH, AFF_AMMOBUILD) && --GET_AMMOBOX_TIME_TO_COMPLETION(PROG) < 1) {
         if (GET_AMMOBOX_TIME_TO_COMPLETION(PROG) <= -2) // --(-1) = -2; prevents penalizing people who ace the test.
-          send_to_char("You seem to have messed up the batch of ammo.\r\n", CH);
+          (i = rand(1,9)) {
+            switch(i) {
+              case 1:
+                send_to_char("You seem to have messed up the batch of ammo.\r\n", CH);
+                break;
+              case 2:
+                send_to_char("You swear loudly as a gunshot rings out because you accidentally fired the round.\r\n", CH);
+                break;
+              case 3:
+                send_to_char("The poor-quality metal shavings you're trying to turn into a bullet disintegrate. you curse the entire lineage of the fixer who gave you these materials\r\n", CH);
+                break;
+              case 4:
+                send_to_char("Somehow, Chinese comes unbidden to your lips. You think you said something unspeakable about a dog - regardless, the bullet is now tainted. You discard it.\r\n", CH);
```

Comment:
I hate to be that guy, but what do the people who have skill in Chinese get for this one? :)